kong/serf.lua

```diff
@@ -35,6 +38,7 @@ function Serf:invoke_signal(signal, args, no_rpc, full_error)
   end
   local rpc = no_rpc and "" or "-rpc-addr="..self.config.cluster_listen_rpc
   local cmd = string.format("%s %s %s %s", self.config.serf_path, signal, rpc, tostring(args))
+  ngx_log(DEBUG, cmd)
```

Comment:
I think we need some context here, aka prefix the command with some kind of a `[serf] running command:` prefix, as we do all around the place.